db_manager/augur_manager.py

```diff
@@ -232,51 +234,153 @@ def project_list_query(self):
+        df_lower_repo_names["rg_name"] = df_lower_repo_names["rg_name"].apply(str.lower)
+        self.org_name_to_repos_dict = df_lower_repo_names.groupby("rg_name")["repo_id"].apply(list).to_dict()
+        self.org_names = list(self.org_name_to_repos_dict.keys())
 
-        # generating dictionary with the org name as the key and the git repos of the org in a list as the value pair
-        self.org_dict = df_search_bar.groupby("rg_name")["repo_git"].apply(list).to_dict()
+        # create a dictionary that maps the github url to the repo_id in padres
+        df_repo_git_id = df_search_bar.copy()
+        df_repo_git_id = df_repo_git_id[["repo_git", "repo_id"]]
+        self.repo_git_to_repo_id = pd.Series(df_repo_git_id.repo_id.values, index=df_repo_git_id["repo_git"]).to_dict()
 
         # making first selection for the search bar
-        self.search_input = self.entries[0]
+        self.initial_search_option = self.multiselect_options[0]
+        logging.debug(f"MULTISELECT_FINISHED")
 
-        logging.debug("Search lists returned and set")
+    def repo_git_to_id(self, git):
+        """Getter method for dictionary
+        that converts a git URL to the respective
+        repo_id in the source db.
 
-    def get_search_input(self):
-        if self.search_input is not None:
-            return self.search_input
-        else:
-            r = self.project_list_query()
-            return self.search_input
+        Args:
+            git (str): URL of repo
 
-    def get_all_entries(self):
-        if self.all_entries is not None:
-            return self.all_entries
-        else:
-            r = self.project_list_query()
-            return self.all_entries
+        Returns:
+            int: repo_id of the URL in the source DB.
+        """
+        return self.repo_git_to_repo_id.get(git)
 
-    def get_org_dict(self):
-        if self.org_dict is not None:
-            return self.org_dict
-        else:
-            r = self.project_list_query()
-            return self.org_dict
+    def org_to_repos(self, org):
+        """Returns the list of repos in an org.
+
+        Args:
+            org (str): Github org name
+
+        Returns:
+            [int] | None: repo_ids or None
+        """
+        return self.org_name_to_repos_dict[org]
+
+    def is_org(self, org):
+        """Checks if org name in set of known org names
+
+        Args:
+            org (str): name of org
+
+        Returns:
+            bool: whether org name is in orgs
+        """
+        return org in self.org_names
+
+    def initial_multiselect_option(self):
+        """Getter method on first multiselect option
+
+        Returns:
+            dict(value, label): first thing the multiselect will represent on startup
+        """
+        return self.initial_search_option
 
-    def get_repo_dict(self):
-        if self.repo_dict is not None:
-            return self.repo_dict
+    def get_multiselect_options(self):
```

Comment:
(this comment is not specific to this line) Does this still make it where the orgs come first then repos?

Reply:
yep! orgs are mapped to a list of repos when selected under the hood.